python/paddle/fluid/layers/nn.py

```diff
@@ -3265,3 +3265,71 @@ def autoincreased_step_counter(counter_name=None, begin=1, step=1):
         counter.stop_gradient = True
 
     return counter
+
+
+def save(x, file_path, overwrite=True):
+    """
+    Saves a variable as a file.
+
+    Args:
+        x(variable): The Tensor/LoDTensor to be saved.
+        file_path(str): The file path where the variable will be saved.
+        overwrite(bool): Whether or not cover the given file when it has already 
+            existed. If it's set 'False' and the file is existed, a runtime 
+            error will be thrown. 
+    """
+    helper = LayerHelper("save", **locals())
+    helper.append_op(
+        type="save",
+        inputs={"input": x},
+        outputs={},
+        args={"file_path": file_path,
+              "overwrite": overwrite})
+
+
+def save_combine(x, file_path, overwrite=True):
+    """
+    Saves a variable as a file.
+
+    Args:
+        x(list): A list of Tensor/LoDTensor to be saved together in a single file.
+        file_path(str): The file path where variables will be saved.
+        overwrite(bool): Whether or not cover the given file when it has already 
+            existed. If it's set 'False' and the file is existed, a runtime 
+            error will be thrown. 
+    """
+    helper = LayerHelper("save_combine", **locals())
+    helper.append_op(
+        type="save_combine",
+        inputs={"input": x},
+        outputs={},
+        args={"file_path": file_path,
+              "overwrite": overwrite})
+
+
+def load(out, file_path):
+    """
+    Args:
+        out(variable): The variable to be read from the disk file.
+        file_path(str): The path of the disk file.
+    """
+    helper = LayerHelper("load", **locals())
+    helper.append_op(
+        type="load",
+        inputs={},
+        output={"Out": out},
+        args={"file_path": file_path})
+
+
+def load_combine(out, file_path):
```

Comment:
need better doc about the difference between save/load and save/load_combine

Reply:
The main difference between save/load and combine save/load is that the combine save can take a list of variables and dump all of them into a single file, while normal save can only save one variable to one file.
You are right, I can make the difference clearer.